.github/workflows/publish_AUR.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will generate a distribution and upload it to PyPI
+
+name: aur-publish
```

Comment:
this one should be removed, its done as part of ovos-binary-shop now

Reply:
#130